openlibrary/catalog/get_ia.py

```diff
@@ -32,7 +35,8 @@ def bad_ia_xml(ia):
     # need to handle 404s:
     # http://www.archive.org/details/index1858mary
     loc = ia + "/" + ia + "_marc.xml"
-    return '<!--' in urlopen_keep_trying(base + loc).read()
+
+    return '<!--' in urlopen_keep_trying(IA_DOWNLOAD_URL + loc).read()
 
 def get_marc_ia_data(ia, host=None, path=None):
```

Comment:
especially confusing since this clobbers `ia` the variable imported above

Reply:
great catch, this ia is supposed to be an ocaid -- I didn't change it earlier because these are (mostly?) deprecated methods and I'm not sure that they are even used, but this naming conflict is a great reason to tidy it up, thanks!